services/ai-navigator-app/0.1.0/ai-navigator-app.yaml

```diff
@@ -0,0 +1,52 @@
+# TODO: Should these be separate files?
+---
+apiVersion: v1
```

Comment:
> Is there a way to apply the catalog from this branch on a cluster?

You could apply this app to the daily if you clone the git repo and just commit these files in the same location they are in this repo. happy to help with this if you want. 

or there’s a CLI flag you can use to point to another repo `—kommander-applications-repository` during install.

Reply:
I assume by that you mean our cluster internal gitea instance? If so, I think I can figure out how to access this :)

Reply:
yep exactly! once you commit these files you should see the `ClusterApp` show up